e2e-testing/cypress/functions/GeneralSettingsFunction.js

```diff
@@ -0,0 +1,45 @@
+import GeneralSettingsLocator from "../locators/GeneralSettingsLocator";
+
+class GeneralSettingsFunctions{
+    // navigate to general settings
+    open_general_settings_page(){
+        cy.get(GeneralSettingsLocator.settingsBtn).click();
+    }
+
+    // edit deployment name
+    type_deployment_name(deploymentName){
+        cy.wait(1000);
+        cy.get(GeneralSettingsLocator.deploymentNameField).clear().type(deploymentName).should('have.value', deploymentName);
```

Comment:
Please change this so that the field name is not cleared. The edit should be appending the word "-Automated" to the existing deployment name. Check the ticket for more information.

Reply:
Sure. Thank you for this feedback.